share/functions/help.fish

```diff
@@ -141,6 +141,18 @@ function help --description 'Show help for the fish shell'
         if type -q cygpath
             set page_url file://(cygpath -m $__fish_help_dir)/$fish_help_page
         end
+
+        # For WSL, provide full path to the file
+        if string match -qr 'cmd.exe$' -- $fish_browser[1]
+            set requires_windows_path 1
+        else if string match -qr 'www-browser' -- $fish_browser[1]
+            if string match -qr 'wslview' (realpath www-browser)
+                set requires_windows_path 1
+            end
+        end
+        if set -q requires_windows_path[1]
+            set page_url (string join \\ '\\\wsl$' $WSL_DISTRO_NAME $__fish_help_dir $__fish_help_page)
```

Comment:
But will the help dir not have unix-style "/"?

Reply:
I admit I'm not 100% familiar with the excentricities of windows' path handling.

Reply:
I was surprised to find that Windows allows you to use it as a path separator, so I didn't bother adding code to replace them with backslashes.